tests/cloudinit/fedora.cloudinit

```diff
@@ -1,42 +1,24 @@
 #!/bin/bash
 
-if [ "@DISABLE_OVERLAY@" = true ] ; then
-  # Workaround for SELinux: do not use overlay fs
-  /sbin/rmmod overlay || true
-  mv /lib/modules/`uname -r`/kernel/fs/overlayfs/overlay.ko.xz /root/overlay.ko.xz-disabled
-fi
-
-if [ "@DISABLE_SELINUX@" = true ] ; then
-  /usr/sbin/setenforce 0
-fi
-
-cat > /var/tmp/rkt-test.sh <<TESTEOF
-#!/bin/bash
-
 set -e
 set -x
 
-# Sometimes journald does not work well on old versions
-exec > >(tee -a "/var/tmp/rkt-test.log") 2>&1
+# ---pretend-input-tty (with 3 dashes) is an undocumented feature of parted
+sudo parted /dev/xvda ---pretend-input-tty resizepart 1 yes 100%
+sudo resize2fs /dev/xvda1
 
-groupadd rkt
-gpasswd -a fedora rkt
+sudo chmod 755 /home/fedora
 
-dnf -y -v update
-dnf -y -v groupinstall "Development Tools"
-dnf -y -v groupinstall "C Development Tools and Libraries"
-dnf -y -v install wget squashfs-tools patch glibc-static gnupg golang libacl-devel file systemd-devel
+#sudo dnf -y -v update
```

Comment:
You could just delete this line.

Note that it was removed for all versions of Fedora.